tests/test_config.cc

```diff
@@ -40,7 +40,7 @@ TEST_CASE("Test config", "[config]"){
   // old ArgManager in cl namespace
   {
 
-    emp::vector<std::string> arguments = {"--dir", "/some_path"};
```

Comment:
I don't know why these unrelated changes are here... I think these have already been merged in on PR #239 . We'll need to double check and maybe fix the Git history.

Reply:
(my bad on this one)